src/components/ActionButtons/BragButton.js

```diff
@@ -15,20 +16,26 @@ const StyledButton = styled(ButtonBase)`
 const BUTTON_LABEL = 'Brag about my efforts';
 const UPDATED_BUTTON_LABEL = 'Brag more';
 
-function BragButton({ role, ...props }) {
+function BragButton({ quoteID = 1, role, ...props }) {
```

Comment:
I think most likely we will change IDs to something different than numeric values be it some proper uuids or funny string combinations. Therefore I would go with marking this prop as required and not providing any default value.